src/button/button-contributions.ts

```diff
@@ -31,6 +31,13 @@ export const isSiteSuitable = (): boolean => {
     return allowedApps.includes(appName);
 }
 
+/**
+ * Checks if the current site is a Gitpod instance.
+ */
+export const isSiteGitpod = (): boolean => {
+  return !!document.head.querySelector("meta[name=Gitpod]");
+}
+
```

Comment:
this doesn't belong here. We are not contributing a button to Gitpod. Let's keep the gitpod dashboard integration completely separate.

Reply:
Dealt with it in 32360975ed64b9e833b341f738afe9784fe1ceed.